### ISSUE TITLE
What is in scope

### ISSUE THREAD

**Issue by ekr** [bdetwiler/draft-detecting-unwanted-location-trackers#119]
I want to make sure we agree on what we are trying to do. Here's my understanding:

  -  Standardize the protocols used by (1) the tag to communicate with finding devices (2) the tag to communicate with devices when apparent unwanted tracking is detected (e.g., to cause it to beep).
   - Not standardize the protocols use by the finding device to report to the tracking network (e.g., to upload to iCloud), the protocols used to initialize the tag, or the protocols used by the tag owner to query the network for location.

One way to phrase this is that it ought to be enough to allow you to build your own tag that would work with the network, assuming it had the right keying material.

Does that sound right? If so, I can provide some text to try to sharpen this

**Comment by ekr** [comment]
No, this does not match my expectation. I believe the entire system including the format of the payload needs to be defined in order to permit security analysis of the system as a whole. I suppose it's possible that we might be able to describe the abstract contents minus the encoding, but i think the contents need to be specified, not just "here's an opaque blob plus some security and privacy requirements".

It's unfortunate that we didn't manage to flesh this out in the meeting.

**Comment by ekr** [comment]
I don't think that this works because we still won't be able to say anything meaningful about the security and privacy properties of anything which doesn't use the reference design. ISTM that the point of designing something in this space is so that we can say "if you do what it says in this spec, then you will have the following properties". That doesn't mean that there's no room for individual variation, but it needs to be straightforward to evaluate whether an implementation meets a given requirement, and that's not the case if it's an opaque blob with some overall requirements.

**Comment by tfpauly** [comment]
@ekr I think I'm on the same page as you for the protocol needing to specify the full advertisement (both the unencrypted and encrypted parts). Looking at the charter definition for the program of work section 1, what specific parts are missing? Maybe just getting your suggested text would be easiest.

> 1. Develop a standards-track protocol ("DULT protocol") between tracking accessories and nearby devices, which may:
> - Specify requirements and a baseline algorithm for determination of unwanted tracking
> - Allow a tracking accessory to identify & advertise its presence when in a detectable mode, using standard formats defined for the underlying transports (e.g., Bluetooth, Near Field Communication, etc.)
> - Allow a nearby device to trigger behavior on an unwanted tracking accessory to aid in determining its physical location
> - Allow nearby devices to fetch additional information about a tracker accessory, including such things as tracker image asset(s) and physical disablement instructions
> - Add privacy and security requirements for all messages used for advertisement, interactions with crowdsourcing networks, and owners of accessories

**Comment by tfpauly** [comment]
@ekr how about this modification to the list? Trying to work it in without sounding awkward.

> 1. Develop a standards-track protocol ("DULT protocol") between tracking accessories and nearby devices, which will:
> - Specify requirements and a baseline algorithm for determination of unwanted tracking
> - Specify complete message formats for accessories to advertise their presence to nearby devices, for one or more underlying transports (e.g., Bluetooth, Near Field Communication, etc.)
> - Allow nearby devices to trigger behavior on an unwanted tracking accessory to aid in determining its physical location
> - Allow nearby devices to fetch additional information about a tracker accessory, including such things as tracker image asset(s) and physical disablement instructions
> - Define privacy and security requirements for all messages used for advertisement, interactions with crowdsourcing networks, and owners of accessories
